src/status_im/utils/handlers.cljs

```diff
@@ -82,8 +82,8 @@
 
 (def default-interceptors
   [debug-handlers-names
-   (when js/goog.DEBUG
-     check-spec)
+   (when false #_js/goog.DEBUG
```

Comment:
RIP spec

Reply:
I will add it back in this PR, but do you plan to remove it?

Reply:
yes soon